sei-incentivized-testnet/README.md

```diff
@@ -0,0 +1,128 @@
+# Becoming A Validator on atlantic-1
+
+## Quick Links
+
+Genesis: [Published](https://github.com/sei-protocol/testnet/raw/main/sei-incentivized-testnet/genesis.json)
+
+Initial addrbook: [Published](https://github.com/sei-protocol/testnet/raw/main/sei-incentivized-testnet/addrbook.json)
+
+Seed node: `df1f6617ff5acdc85d9daa890300a57a9d956e5e@sei-atlantic-1.seed.rhinostake.com:16660`
+
+| Additional documentation is available on the [seinetwork.io website](https://docs.seinetwork.io/nodes-and-validators/seinami-incentivized-testnet/joining-incentivized-testnet).
+
+## Hardware / Software Requirements
+
+### Minimum
+
+- 3.2 GHz 4 Core CPU
+- 8 GB RAM
+- 100 GB NVME SSD
+
+### Recommended
+
+- 4.2 GHz 6 Core CPU
+- 16 GB RAM
+- 500 GB NVME SSD
+
+### Operating System
+
+- Linux (x86_64) or Linux (amd64)
+- Recommended Arch Linux, but any modern distribution will work.
+
+### Dependencies
+
+> Prerequisite: go1.18+ required.
+
+- Arch Linux: `pacman -S go`
+- Ubuntu: `sudo snap install go --classic`
+- Source: `wget https://go.dev/dl/go1.18.4.linux-amd64.tar.gz && sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.18.4.linux-amd64.tar.gz`. Be sure to also add `export PATH=$PATH:/usr/local/go/bin` to your .bashrc/.zshrc file.
+
+> Prerequisite: git.
+
+- Arch Linux: `pacman -S git`
+- Ubuntu: `sudo apt-get install git` (should be installed by default)
+
+> Optional requirement: GNU make.
+
+- Arch Linux: `pacman -S make`
+- Ubuntu: `sudo apt-get install make` or `sudo apt-get build-essential`
+
+## Seid Installation Steps
+
+### Clone git repository
+
+```bash
+git clone https://github.com/sei-protocol/sei-chain
+cd sei-chain
+git checkout <tag-name>
+# Current testnet tag is 1.0.6beta
+make install
+```
+
+## Generate Keys
+
+- `seid keys add [key_name]`
+
+- `seid keys add [key_name] --recover` to regenerate keys with your mnemonic
+
+- `seid keys add [key_name] --ledger` to generate keys with ledger device
+
+## Validator Setup Instructions
+
+- Install seid binary
+- Initialize node: `seid init <moniker> --chain-id=atlantic-1`
+- Download the Genesis file: `curl -s https://github.com/sei-protocol/testnet/raw/main/sei-incentivized-testnet/genesis.json > $HOME/.sei/config/genesis.json`
+- Edit the minimum-gas-prices in ${HOME}/.sei/config/app.toml: `sed -i 's/minimum-gas-prices = ""/minimum-gas-prices = "0.01usei"/g' $HOME/.sei/config/app.toml`
```

Comment:
nit (this is done for you already in sei-chain): https://sourcegraph.com/github.com/sei-protocol/sei-chain@master/-/blob/cmd/seid/cmd/root.go?L332:9

Reply:
Good catch.  I'm glad the Sei team adds this automatically. I actually copied this from the default README at the root of this repo, maybe it was there just from the original devnet.